Package.swift

```diff
@@ -1,4 +1,4 @@
-// swift-tools-version:5.6
+// swift-tools-version:5.7
```

Comment:
What's the reason for the version bump? (I often avoid version bumping to keep support for older OSes)

Reply:
5.7 is to bring it in line with the official SPM branch. The minimum requirement was recently brought up from 5.6 to 5.7 build tools.

Reply:
Ah ok, if you set it to 5.6 does it still build?

Reply:
Yes, the only issue is the `package-description-version` which I have since made dynamic 🎉